practitioner-service/src/test/java/uk/ac/ox/ndph/mts/practitioner_service/repository/FhirContextWrapperIntegrationTest.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.ox.ndph.mts.practitioner_service.repository;
+
+import org.hl7.fhir.r4.model.Bundle;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class FhirContextWrapperIntegrationTest {
```

Comment:
1. What makes this an integration test (btw, I don't think that PractitionerIntegrationTest are it either)?
2. Would something like this work?
```java
@SpringBootTest(properties = { "fhir.uri=something" })
```

Reply:
Ah yes... I thought this was going to be an integration test, but turns out it was half-baked! I've renamed the class, and finished implementing the test.

I thought the same thing when I saw PractitionerIntegrationTests.java, but after pondering for a while I think the original author was testing the boundary between the code we control and the code we don't (i.e. the Spring framework). They also seem to take longer than the other unit tests, so could be they were trying to highlight that aspect too...